src/vtvalidate.cpp

```diff
@@ -100,14 +122,19 @@ struct AsyncHelloWorker : Nan::AsyncWorker {
         callback->Call(argc, static_cast<v8::Local<v8::Value>*>(argv));
     }
 
+    // explicitly use the destructor to clean up
+    // the persistent tile ref by Reset()-ing
+    ~AsyncValidateWorker() {
+        tile_.Reset();
+    }
+
     std::string result_;
-    const bool louder_;
+    vtzero::data_view data;
```

Comment:
Do we need to store a data_view? I think we can use the `ref_buffer` and call 
`data_view data(node::Buffer::Data(buffer), node::Buffer::Length(buffer))` as we need it?

Reply:
@flippmoke Then what's the point of setting the Persistent `tile_` var to `buffer` if we're going to set the data_view to the v8::Local object?

Reply:
Is this what you're suggesting? https://gist.github.com/GretaCB/a98d8d433678ed0e2ae443d999b68515